solana-programs/anchor/audius-data/tests/audius-data.ts

```diff
@@ -1115,8 +1027,158 @@ describe("audius-data", function () {
     });
   });
 
+  it("delegate creates a track (manage entity) + all validation errors", async function () {
+    // create user and delegate
+    const delegate = await testCreateUserDelegate({
+      adminKeypair,
+      adminStorageKeypair: adminStorageKeypair,
+      program,
+      provider,
+      ...getURSMParams(),
+    });
+
+    // create track with delegate
+    const trackMetadata = randomCID();
+    const trackID = randomId();
+
+    await expect(
```

Comment:
would be clearer if this were broken out into a separate `it('delegated create track action fails when delegate account PDA is not provided')` etc etc for the other cases.

Reply:
i think this actually fits in pretty smoothly here and reduces more code duplication